docs/fsp_web_auth.md

```diff
@@ -0,0 +1,18 @@
+# Instructions for FSP Web Auth
+
+The FSP must ensure that when creating their web login, the web login redirects the user to a URI that is recognizable by the app so that the auth token may be extracted.
+
+The URI that the login page must redirect the user to upon successful login is of the format:
+
+pisp://...?token={auth_token}
```

Comment:
Maybe can put this under a text section?

Something like
```
pisp://...?token={auth_token}
```

Reply:
Agreed